product_docs/docs/biganimal/release/using_cluster/01_postgres_access/index.mdx

```diff
@@ -109,32 +109,37 @@ Provision your cluster before configuring IAM for Postgres.
     1. Select **Create Cluster** or **Save**.
     !!!note
     To turn on IAM authentication using the CLI, see [Using IAM authentication on AWS](/biganimal/latest/reference/cli/using_features/#iam-authentication-cli-commands).
-1. In AWS, get the ARN of each IAM user requiring database access. In the AWS account connected to BigAnimal, use AWS Identity and Access Management (IAM) to perform user management. See the [AWS documentation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_users_manage.html). 
+1. From your cloud provider, get the user name of each IAM user requiring database access. In the cloud account connected to BigAnimal, use Identity and Access Management (IAM) to perform user management. 
 
-1. In Postgres, if the IAM role doesn’t exist yet, run this Postgres command:
+1. In Postgres, if the IAM role doesn’t exist yet, use the `CREATE ROLE` command. For example, for AWS, use:
 
     ```
     CREATE ROLE "iam_aws";
     ```
 
-1. For each IAM user, run this Postgres command:
+1. For each IAM user, run the `CREATE USER` Postgres command. For example, for AWS, use:
 
     ```
     CREATE USER "<ARN>" IN ROLE iam_aws;
     ```
 
+    Where <ARN> is the Amazon resource name. (For Azure, use the user principal name. For GCP, use the email address.)
+
 ### Logging in to Postgres using IAM credentials
 
-If IAM integration is configured for your cluster, you can log in to Postgres using your AWS Amazon Resource Name (ARN) and access key. Using this ARN + access key combination allows you to connect to your Postgres database using your AWS IAM standard credentials.
+If IAM integration is configured for your cluster, you can log in to Postgres using your cloud credentials. Alternatively, you can use your token instead of your password. Logging in either way allows you to connect to your Postgres database using your cloud account's IAM standard credentials. 
+
+For either method, you must first authenticate to your cloud service provider IAM to get your password or token.
 
 !!! Note 
     You can continue to log in using your Postgres username and password. However, doing so doesn’t provide IAM authentication even if this feature is configured.
 
-1. Using your AWS CLI or Cloud shell, obtain your ARN and access key. For guidance on obtaining your ARN and access key, see [Managing access keys for IAM users](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html).
+1. Get your credentials for your IAM-managed cloud account.
+    - For AWS, your password is your access key (access key id + secret access key). To get your access key, see [get-access-key-info](https://docs.aws.amazon.com/cli/latest/reference/sts/get-access-key-info.html) To get your authorization token, see [get-authorization-token](https://docs.aws.amazon.com/cli/latest/reference/ecr-public/get-authorization-token.html).
```

Comment:
@edbmanniwood is there a colon in between the access key id and secret access key?

Reply:
Hello, @amrita-suresh and @ebgitelman ! Yes, AWS users will want to place a colon between their aws access key ID and their aws secret access key (<aws_access_key_id>:<aws_secret_access_key>) and then they can use the resulting string of characters as their password.

Reply:
```suggestion
    - For AWS, your password is your access key (in the form <access key id>:<secret access key>). To get your access key, see [get-access-key-info](https://docs.aws.amazon.com/cli/latest/reference/sts/get-access-key-info.html) To get your authorization token, see [get-authorization-token](https://docs.aws.amazon.com/cli/latest/reference/ecr-public/get-authorization-token.html).
```